models/public/yolo-v1-tiny-tf/yolo-v1-tiny-tf.md

```diff
@@ -0,0 +1,96 @@
+# yolo-v1-tiny-tf
+
+## Use Case and High-Level Description
+
+YOLO v1 Tiny is a real-time object detection model from TensorFlow JS\* framework. This model was pretrained on VOC\* dataset with 20 classes.
+
```

Comment:
Since this is a reuploaded model, you need to document how to convert it to the form it's downloaded in.